modules/checklist/assets/js/components/checklist-item.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardContent from '@elementor/ui/CardContent';
+import CardActions from '@elementor/ui/CardActions';
+import Typography from '@elementor/ui/Typography';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import ListItemIcon from '@elementor/ui/ListItemIcon';
+import ListItemText from '@elementor/ui/ListItemText';
+import Collapse from '@elementor/ui/Collapse';
+import ChevronDownIcon from '@elementor/icons/ChevronDownIcon';
+import RadioButtonUncheckedIcon from '@elementor/icons/RadioButtonUncheckedIcon';
+import CardMedia from '@elementor/ui/CardMedia';
+import Link from '@elementor/ui/Link';
+
+function CheckListItem( props ) {
+	const { id, title, description, link, CTA } = props.step,
+		{ expandedIndex, setExpandedIndex } = props,
+		[ expanded, setExpanded ] = useState( false );
+
+	const handleExpandClick = () => {
+		setExpanded( ! expanded );
```

Comment:
This needs adjustment: When you take the index (number), the `!` operator will turn it to false, and then the next time it will turn to true.
```suggestion
		setExpanded( expanded !== null ? null : props.index );
```

Reply:
If you want to keep using `setExpanded`, then she must keep the state, no?

Reply:
My bad, I meant to write `setExpandedIndex` from the props
Good catch!